StreamChatClient_v3/Controllers/ChannelListController_Tests.swift

```diff
@@ -116,6 +116,60 @@ class ChannelListController_Tests: StressTestCase {
         // Completion should be called with the error
         XCTAssertEqual(completionCalledError as? TestError, testError)
     }
+
+    // MARK: - Test controller actions
+
+    func test_deleteChannel_callsChannelUpdater() {
+        let cid: ChannelId = .unique
+        var completionCalled = false
+
+        controller.deleteChannel(cid: cid) { error in
+            XCTAssertNil(error)
+            completionCalled = true
+        }
+
+        XCTAssertFalse(completionCalled)
+        env.channelUpdater?.deleteChannel_completion?(nil)
```

Comment:
You should test that the parameter you call the completion block with is the parameter that the called receives. Check other tests where we use `TestError` for situations like this.